tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -652,18 +614,18 @@ describe(__filename, () => {
         undefined,
         loadAddonsByAuthors({
           addons: Object.values(addons),
-          authorUsernames: ['fakeUsername'],
+          authorIds: [randomAuthorId1],
           count: Object.values(addons).length,
           pageSize: EXTENSIONS_BY_AUTHORS_PAGE_SIZE,
         }),
       );
 
-      expect(getAddonsForUsernames(state, ['test2', 'no-addons-user'])).toEqual(
-        [
-          createInternalAddon(addons.firstAddon),
-          createInternalAddon(addons.secondAddon),
-        ],
-      );
+      expect(
+        getAddonsForAuthorIds(state, [fakeAuthorTwo.id, randomAuthorId2]),
```

Comment:
I find this very hard to follow. How am I supposed to know to expect that I'll get back `addons.firstAddon` and `addons.secondAddon` when I call `getAddonsForAuthorIds` passing in `fakeAuthorTwo.id` and `randomAuthorId2`? There's nothing in the test that makes that clear, so it's very difficult to know that this test is actually set up correctly for the expected outcome.